src/modules/vsf-storyblok-module/components/index.ts

```diff
@@ -1,14 +1,14 @@
 export const components = {
   debug: () => import('./defaults/Debug.vue'),
-  page: () => import('./defaults/Page.vue'),
-  block: () => import('./defaults/Block.vue'),
+  page: () => import(/* webpackChunkName: "vsf-storyblok-common-components" */ './defaults/Page.vue'),
+  block: () => import(/* webpackChunkName: "vsf-storyblok-common-components" */ './defaults/Block.vue'),
   category_description: () => import('./defaults/Block.vue'),
   product_description: () => import('./defaults/Block.vue'),
```

Comment:
I think it is better to pack all Storyblok resources into one bundle since we can't predict which will actually be used on the page and there are a lot of different components on the page.

We can exclude ones triggering heavy resources load.